packages/terra-show-hide/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra Show Hide
+
+
+[![NPM version](https://badgen.net/npm/v/terra-show-hide)](https://www.npmjs.org/package/terra-abstract-modal)
+[![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.org/cerner/terra-core)
+
+Show Hide component that will show a preview of content or show the full content with the click of a button.
```

Comment:
Nitpick: I feel like the component says you will either get a preview or you will get the full content makes it seem like they don't work together well. I think this would make it more clear.

'Show Hide Component that will show a preview of content and then expand it with a Show More button"

Reply:
I could see how that description could be confusing: https://github.com/cerner/terra-core/pull/1956/commits/de9fc6e3e930fe444d281a4b584843906853943f